packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -949,15 +951,27 @@ export class PipelineEditor extends React.Component<
    * if there are invalid properties. If there are none,
    * returns null.
    */
-  validateProperties(node: any): string {
+  async validateProperties(node: any): Promise<string> {
+    const notebookValidationErr = await this.app.serviceManager.contents
+      .get(node.app_data.filename)
```

Comment:
Quick parallel dev note here:

If #861 is merged first then this will need to be updated from `node.app_data.filename` to `PipelineService.getWorkspaceRelativeNodePath(this.widgetContext.path, node.app_data.filename);`

If this is merged first then it will need to be added to #861 prior to it's merge